spec/support/authentication_helper.rb

```diff
@@ -1,27 +1,27 @@
 module AuthenticationHelper
-  # def sign_in_as(user)
-    # mock_github_user_auth!(user)
+  def sign_in_as(user)
+    mock_github_user_auth!(user)
 
-    # visit root_path
-    # find(:css, '#sign-in').click
+    visit root_path
+    find(:css, '#sign-in').click
 
-    # click_on 'Sign in with Github'
-  # end
+    click_on 'Sign in with Github'
+  end
 
-  # def mock_github_user_auth!(user)
-    # OmniAuth.config.mock_auth[:github] = {
-      # "provider" => "github",
-      # "uid" => "123456",
-      # "info" => {
-        # "nickname" => user.nickname,
-        # "email" => user.email,
-        # "name" => user.first_name,
-        # "image" => "http://www.s3link.com"
-      # },
-      # "credentials" => {
-        # "token" => "12345"
-      # }
-    # }
-  # end
+  def mock_github_user_auth!(user)
+    OmniAuth.config.mock_auth[:github] = OmniAuth::AuthHash.new({
+      "provider" => "github",
+      "uid" => "123456",
+      "info" => {
+        "nickname" => user.nickname,
+        "email" => user.email,
+        "name" => user.first_name,
+        "image" => "http://www.s3link.com"
+      },
+      "credentials" => {
+        "token" => "12345"
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.